sourmash/minhash.py

```diff
@@ -290,39 +290,41 @@ def get_mins(self, with_abundance=False):
         """Return list of hashes or if ``with_abundance`` a list
         of (hash, abund).
         """
+        mins = self.hashes
+        if not with_abundance:
+            return mins.keys()
+        return mins
+
+
+    @deprecated(deprecated_in="3.5", removed_in="5.0",
```

Comment:
note I had said "removed in 5.0" here... do we want to change that to 4.0 in stable?

Reply:
Both `.get_mins` and `.get_hashes` continue working as before, I didn't change the deprecations (is this "GitHub rendering the diffs in a confusing way" issue?)

Reply:
yes, how odd, when I look at it now it's clear, but before it looked like it had been removed.